scripts/release.sh

```diff
@@ -81,25 +87,102 @@ create_gh_release() {
 }
 
 build_docker_image() {
+  echo ">>> Building the operator docker image ..."
   docker build -t splunk-otel-instrumentation-java .
   docker tag splunk-otel-instrumentation-java quay.io/signalfx/splunk-otel-instrumentation-java:$release_tag
-  docker tag splunk-otel-instrumentation-java quay.io/signalfx/splunk-otel-instrumentation-java:$major_version
+  docker tag splunk-otel-instrumentation-java quay.io/signalfx/splunk-otel-instrumentation-java:v$major_version
   docker tag splunk-otel-instrumentation-java quay.io/signalfx/splunk-otel-instrumentation-java:latest
 }
 
+login_to_quay_io() {
+  echo ">>> Logging into quay.io ..."
+  docker login -u $QUAY_USERNAME -p $QUAY_PASSWORD quay.io
+}
+
 publish_docker_image() {
+  echo ">>> Publishing the operator docker image ..."
   docker push quay.io/signalfx/splunk-otel-instrumentation-java:latest
   docker push quay.io/signalfx/splunk-otel-instrumentation-java:$release_tag
-  docker push quay.io/signalfx/splunk-otel-instrumentation-java:$major_version
+  docker push quay.io/signalfx/splunk-otel-instrumentation-java:v$major_version
 }
 
-login_to_quay_io() {
-  docker login -u $QUAY_USERNAME -p $QUAY_PASSWORD quay.io
+# TODO: add it to o11y-gdi env vars
+GITHUB_BOT_GPG_KEY_ID="A2581E631864FD064"
```

Comment:
Saving this for later, or do you want to do it as part of this PR?

Reply:
Nah, that can be done later, in a separate PR. I don't have admin access to o11y-gdi so I'll probably need to wait for that a bit.